kimchi/src/circuits/polynomials/range_check/gate.rs

```diff
@@ -269,7 +269,7 @@ struct LookupEnvironmentData<F: FftField> {
 // computing the dummy lookup value, creating the combined lookup table, computing the sorted plookup
 // evaluations and the plookup aggregation evaluations.
 // Note: This function assumes the cs contains a lookup constraint system.
-fn set_up_lookup_env_data<F: PrimeField, G: KimchiCurve<ScalarField = F>>(
+fn set_up_lookup_env_data<G: KimchiCurve<ScalarField = F>, F: PrimeField>(
```

Comment:
my question was why pass G here tho? It doesn't look like it's required

Reply:
Right, it only needed it because `ConstraintSystem` that now is generic over field, removed.